index.css

```diff
@@ -388,3 +403,213 @@ label {
   font-weight: 400;
   line-height: 28px;
 }
+
+.list_style {
+  list-style: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .nav1 {
+    display: none;
+  }
+
+  .nav2 {
+    display: block;
+    display: flex;
+    gap: 20px;
+  }
+
+  .nav2 a {
+    font-size: 15px;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  .section1 {
+    display: flex;
+    justify-content: center;
+  }
+
+  .section1 h1 {
+    font-size: 48px;
+    color: #172b4d;
+    line-height: 60px;
+    font-weight: 700;
+    width: 420px;
+  }
+
+  .section1 p {
+    color: #344563;
+    font-size: 13px;
+    line-height: 28px;
+    width: 580px;
+    font-weight: 400;
+    align-items: left;
+    padding: 50px 0 0 0;
+  }
+
+  .works_section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 90%;
+    margin: auto;
+    padding: 20px 0 30px 0;
+  }
+
+  .works_section .article1,
+  .works_section .article2,
+  .works_section .article3,
+  .works_section .article4 {
+    display: flex;
+    justify-content: center;
+    gap: 15px;
+    margin: 20px 0;
+  }
+
+  .article1 .figure1,
+  .article2 .figure1,
+  .article3 .figure1,
+  .article4 .figure1 {
+    display: none;
+  }
+
+  .article1 .figure2,
+  .article2 .figure2,
+  .article3 .figure2,
+  .article4 .figure2 {
+    display: block;
+    display: flex;
+    justify-content: center;
+  }
+
+  .article1div,
+  .article2div,
+  .article3div,
+  .article4div {
+    padding: 0;
+    align-items: left;
+  }
+
+  .about_section_align1 img {
+    padding: 12px;
+    background-color: #f7f7f9;
+    border: 1px solid none;
+    border-radius: 10px;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .article1 .figure2 img,
+  .article2 .figure2 img,
+  .article3 .figure2 img,
+  .article4 .figure2 img {
+    width: 340px;
+    height: 300px;
+    object-fit: cover;
+    object-position: center;
+  }
+
+  .show1 {
+    display: none;
+  }
+
+  .show2 {
+    display: block;
+  }
+
+  .font1 {
+    padding-top: 0;
+    color: #091e42;
+    font-size: 36px;
+    font-weight: 700;
+  }
+
+  .font2 {
+    width: 295px;
+    color: #344563;
+    font-size: 18px;
+    font-weight: 500;
+  }
+
+  .font3 {
+    width: 295px;
+    color: #6b778c;
+    font-size: 18px;
+    font-weight: 400;
+  }
+
+  .article1div p,
+  .article2div p,
+  .article3div p,
+  .article4div p,
+  .about_self_section p {
+    color: #344563;
+    font-size: 13px;
+    line-height: 20px;
+    font-weight: 500;
+  }
+
+  .about_self_section {
+    display: flex;
+    justify-content: center;
+    width: 90%;
+    margin: auto;
+    padding: 70px 0 100px 0;
+  }
```

Comment:
Please adjust the width of your about myself section😀:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/199867682-727c1d93-a7b8-4ce5-b382-e7668761ed41.png)  | ![image](https://user-images.githubusercontent.com/42799579/199867712-c69439fe-db44-4ebc-aaef-1e9398c85c13.png)  |

Notice that in the original the card occupies the whole screen and the spacing of the elements inside it is larger 😉.